main/Main.hs

```diff
@@ -64,11 +61,11 @@ main = do
   -- read PGRST_ env variables
   env <- readEnvironment
 
-  -- read path from commad line
-  opts <- readCLIShowHelp env
+  -- read command/path from commad line
+  (command, confPath) <- ((,) <$> cliCommand <*> cliPath) <$> readCLIShowHelp env
```

Comment:
I like the idea - but maybe it's possible to use `RecordWildCards` here, so something like this?

```haskell
CLI{..} <- readCLIShowHelp env
```

or `NamedFieldPuns` if you want to be more explicit?

```haskell
CLI{cliCommand, cliPath} <- readCLIShowHelp env
```

In both cases you should be able to use `cliCommand` and `cliPath` directly without the `opts` afterwards.